{"uid":"670f87fa807ac0ff","name":"Попытка принять заказ без ID заказа","fullName":"tests.orders.test_accept_order_missing_order_id.TestAcceptOrderMissingOrderId#test_accept_order_missing_courier_id","historyId":"643f9c319b504593f0773f9f18b9404b","time":{"start":1734160179127,"stop":1734160184156,"duration":5029},"status":"failed","statusMessage":"AssertionError: Expected status 400 with 'Недостаточно данных для поиска', but got 404 and {'code': 404, 'message': 'Not Found.'}\nassert (404 == 400)\n +  where 404 = <Response [404]>.status_code","statusTrace":"self = <tests.orders.test_accept_order_missing_order_id.TestAcceptOrderMissingOrderId object at 0x10a987800>\n\n    @allure.title(\"Попытка принять заказ без ID заказа\")\n    def test_accept_order_missing_courier_id(self):\n        order_methods = OrderMethods()\n        courier_methods = CourierMethods()\n        courier_methods.create_courier(\n            COURIER_DATA[\"valid_courier\"][\"login\"],\n            COURIER_DATA[\"valid_courier\"][\"password\"],\n            COURIER_DATA[\"valid_courier\"][\"firstName\"]\n        )\n        login_response = courier_methods.login_courier(\n            COURIER_DATA[\"valid_courier\"][\"login\"],\n            COURIER_DATA[\"valid_courier\"][\"password\"],\n        )\n        login_response.json().get(\"id\")\n        order_data = ORDER_DATA[\"order_data_black\"]\n        create_order_response = order_methods.create_order(order_data)\n        track_number = create_order_response.json().get(\"track\")\n        get_order_response = order_methods.get_order_by_track(track_number)\n        order_id = get_order_response.json()[\"order\"].get(\"id\")\n        accept_order_response = order_methods.accept_order(\"\", order_id)\n        response_json = accept_order_response.json()\n>       assert accept_order_response.status_code == 400 and response_json.get(\"message\") == \"Недостаточно данных для поиска\", (\n            f\"Expected status 400 with 'Недостаточно данных для поиска', but got {accept_order_response.status_code} and {response_json}\"\n        )\nE       AssertionError: Expected status 400 with 'Недостаточно данных для поиска', but got 404 and {'code': 404, 'message': 'Not Found.'}\nE       assert (404 == 400)\nE        +  where 404 = <Response [404]>.status_code\n\ntests/orders/test_accept_order_missing_order_id.py:29: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"feature","value":"Принятие заказа"},{"name":"parentSuite","value":"tests.orders"},{"name":"suite","value":"test_accept_order_missing_order_id"},{"name":"subSuite","value":"TestAcceptOrderMissingOrderId"},{"name":"host","value":"MBP-Dmitrij.lan"},{"name":"thread","value":"6337-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.orders.test_accept_order_missing_order_id"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"670f87fa807ac0ff.json","parameterValues":[]}